packages/driver/src/cy/commands/request.coffee

```diff
@@ -125,6 +125,19 @@ module.exports = (Commands, Cypress, cy, state, config) ->
       if originOrBase = config("baseUrl") or cy.getRemoteLocation("origin")
         options.url = $Location.qualifyWithBaseUrl(originOrBase, options.url)
 
+      ## Make sure the url unicode characters are properly escaped
+      ## https://github.com/cypress-io/cypress/issues/5274
+      try
+        options.url = new URL(options.url).href
+      catch TypeError
```

Comment:
this is not doing filtering on the `catch` - it's only yielding the local error instance as `TypeError`

Reply:
:man_facepalming: at least all the tests pass... @avallete do you want to open up a PR to clean this up, or do you want me to?

Reply:
@brian-mann  Actually my intent here wasn't to filter on the error type because as stated into the [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) conditional `catch` based on the type of the error isn't standard. Also, based on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) the only error that a `new URL` failing can throw is a `TypeError` exception. 

So my intent wasn't to filter the error, I do want to globally catch any error raised on this line, and, since the only one who can occur is `TypeError`, I named the variable accordingly.

@flotwig if you do want me to open a PR with specific filtering like: `catch (e if e instanceof TypeError)`

I can do it, no problems ;)

Reply:
Ah, thank you for clearing that up, we were both scratching our heads trying to figure out the intent here. :sweat_smile:  Opened a PR to clean it up: https://github.com/cypress-io/cypress/pull/5937

Reply:
Nice, sorry for the double work ration for you :stuck_out_tongue: